torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -3014,9 +3130,12 @@ def named_buffers(
         remove all occurrences of the FSDP-specific flattened buffer prefix
         when inside the :meth:`summon_full_params` context manager.
         """
-        in_summon_full_params = self.training_state == TrainingState_.SUMMON_FULL_PARAMS
+        should_clean_name = (
+            self.training_state == TrainingState_.SUMMON_FULL_PARAMS
+            or self._use_orig_params
```

Comment:
why do we need to clean the name when using use_orig_params? Shouldn't the param FQNs be exactly the local param names?

Reply:
There is still nested wrapping (FSDP -> FPW -> module), so I think the names will be unclean.